src/apply.jl

```diff
@@ -39,22 +39,36 @@ end # loop
 # TimeArray <--> Scalar
 for op in [MATH_ALL; COMPARE_DOTONLY]
     @eval begin
-        function ($op){T<:Number,N}(ta::TimeArray{T,N}, var::Number)
+        function broadcast(::typeof($op), ta::TimeArray{T, N},
+                           var::Number) where {T <: Number, N}
             cnames  = [name * string($op) * string(var) for name in ta.colnames]
-            vals = ($op)(ta.values, var)
+            vals = broadcast($op, ta.values, var)
             TimeArray(ta.timestamp, vals, cnames, ta.meta)
         end # function
+
+        # FIXME: make some function allowed dot version only
+        # e.g. `ta > 42` is not allowed, but `ta .> 42` does
+        if Symbol($op) ∉ [$MATH_DOTONLY; $COMPARE_DOTONLY]
+            ($op)(ta::TimeArray{T, N}, var::Number) where {T <: Number, N} =
```

Comment:
hmmm... doesn't work
diff --git a/src/apply.jl b/src/apply.jl
```patch
index e3079a2..13d4fdc 100644
--- a/src/apply.jl
+++ b/src/apply.jl
@@ -55,6 +55,12 @@ for op in [MATH_ALL; COMPARE_DOTONLY]
     end # eval
 end # loop
 
+function broadcast(::typeof(isless), ta::TimeArray{T, N}, var::Number) where {T <: Number, N}
+    cnames = [name * string(<) * string(var) for name in ta.colnames]
+    vals = broadcast(isless, ta.values, var)
+    TimeArray(ta.timestamp, vals, cnames, ta.meta)
+end
+
 # Scalar <--> TimeArray
 for op in [MATH_ALL; COMPARE_DOTONLY]
     @eval begin
```

found that the function list in stacktrace keeping changes:
```julia
julia> isless.(cl, 1)
...
Stacktrace:
 [1] (::##1#2)(::TimeSeries.TimeArray{Float64,1,Date,Array{Float64,1}}) at ./<missing>:0
 [2] broadcast(::Function, ::TimeSeries.TimeArray{Float64,1,Date,Array{Float64,1}}) at ./broadcast.jl:434

julia> isless.(cl, 1)
...
Stacktrace:
 [1] (::##3#4)(::TimeSeries.TimeArray{Float64,1,Date,Array{Float64,1}}) at ./<missing>:0
 [2] broadcast(::Function, ::TimeSeries.TimeArray{Float64,1,Date,Array{Float64,1}}) at ./broadcast.jl:434

julia> isless.(cl, 1)
..
Stacktrace:
 [1] (::##5#6)(::TimeSeries.TimeArray{Float64,1,Date,Array{Float64,1}}) at ./<missing>:0
 [2] broadcast(::Function, ::TimeSeries.TimeArray{Float64,1,Date,Array{Float64,1}}) at ./broadcast.jl:434
```
Is this related to [automatically fusing on dot call](https://docs.julialang.org/en/latest/manual/mathematical-operations/#man-dot-operators-1)?